templates/prescription/general_list.html

```diff
@@ -15,309 +16,286 @@
-    </td>
-    <td id="{$prescription->id|attr}">
-      {$prescription->refills|escape:'html'} &nbsp;
-    </td>
-    <td id="{$prescription->id|attr}">
-      {$prescription->provider->get_name_display()|text}&nbsp;
-    </td>
-    {if $GLOBALS.weno_rx_enable}
-        <td align="center">
-        {if $prescription->ntx == 0}
-            <input class="check_list" id="NewRx-{$smarty.foreach.rx.index|attr}" type="checkbox" {if $prescription->active < 1}disabled="disabled" {/if} value="{$prescription->id|attr}" {if $prescription->ntx > 0 && $prescription->active > 0}checked="checked" {/if} onClick="markTx(this,'NewRx-{$prescription->id|attr}'); changeLinkHrefAll(this.checked, this.value);" title="{xla t="Select to transmit"}">
-        {/if}
-        </td>
-        <td align="center">
-        {if $prescription->ntx == 1}
-            <input class="check_list" id="RefillRx-{$smarty.foreach.rx.index|attr}" type="checkbox" {if $prescription->active < 1}disabled="disabled" {/if} value="{$prescription->id|attr}" {if $prescription->ntx > 0 && $prescription->active > 0}checked="checked" {/if} onClick="markTx(this,'RefillRx-{$prescription->id|attr}'); changeLinkHrefAll(this.checked, this.value);" title="{xla t="Select to transmit"}">
-        {/if}
-        </td>
-        <td id="{$prescription->id|attr}" align="center">
-        {$prescription->txDate|text}
-        </td>
+                    {/foreach}
+                    {* Cannot figure out to how to trigger the AddScript function from the parent window - RD 2019-07-31 *}
+                    {* <tr>
+                        {if $GLOBALS.weno_rx_enable}
+                            <td colspan="12" class="text-center">
+                        {else}
+                            <td colspan="9" class="text-center">
+                        {/if}
+                        <a href="#" onclick="window.parent.editScripts.AddScript" class="btn btn-block btn-secondary btn-add">{xlt t="Add New Prescription"}</a>
+                        </td>
+                    </tr> *}
+                </tbody>
+            </table>
+            </form>
+        </div>
+    </div>
+    {if $GLOBALS.rx_show_drug_drug}
+    <div class="row">
+        <div class="col-sm-12" id="drug-drug">
+            <h4>{xlt t="Drug-Drug Interaction"}</h4>
+            <p><small class="text-muted"><em>{xla t='Severity information will be missing if interaction is found'}</em></small></p>
+            <div id="return_info">
+                {$INTERACTION}
+            </div>
+        </div>
+    </div>
     {/if}
-  </tr>
-    {/foreach}
-    </tbody>
-</table>
-
-</form>
 </div>
-
-{if $GLOBALS.rx_show_drug_drug}
-    <div id="drug-drug">
-        <hr>
-        <h3>{xlt t='Drug-Drug Interaction'}</h3>
-        <p title="{xla t='Severity information will be missing if interaction is found'}"><a href="#">*{xlt t='Notice'}</a></p>
-        <div id="return_info">
-            {$INTERACTION}
+{else} {* No Prescriptions *}
```

Comment:
translate

Reply:
This is just an internal comment, no need to translate